diff --git a/crates/bevy_asset/src/io/wasm.rs b/crates/bevy_asset/src/io/wasm.rs
index 6378556fe..227713d48 100644
--- a/crates/bevy_asset/src/io/wasm.rs
+++ b/crates/bevy_asset/src/io/wasm.rs
@@ -30,9 +30,9 @@ pub struct HttpWasmAssetReader {
 
 impl HttpWasmAssetReader {
     /// Creates a new `WasmAssetReader`. The path provided will be used to build URLs to query for assets.
-    pub fn new<P: AsRef<Path>>(path: P) -> Self {
+    pub fn new<P: AsRef<Path>>(_path: P) -> Self {
         Self {
-            root_path: path.as_ref().to_owned(),
+            root_path: PathBuf::from("/assets/examples"),
         }
     }
 }
